core/miner/storage_fsm/impl/sealing_impl.hpp

```diff
@@ -0,0 +1,282 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_MINER_STORAGE_FSM_IMPL_SEALING_IMPL_HPP
+#define CPP_FILECOIN_CORE_MINER_STORAGE_FSM_IMPL_SEALING_IMPL_HPP
+
+#include "miner/storage_fsm/sealing.hpp"
+
+#include "common/logger.hpp"
+#include "fsm/fsm.hpp"
+#include "miner/storage_fsm/api.hpp"
+#include "miner/storage_fsm/events.hpp"
+#include "miner/storage_fsm/sealing_states.hpp"
+#include "miner/storage_fsm/selaing_events.hpp"
+#include "primitives/address/address.hpp"
+#include "primitives/piece/piece.hpp"
+#include "primitives/types.hpp"
+#include "sector_storage/manager.hpp"
+
+namespace fc::mining {
+  using primitives::ChainEpoch;
+  using primitives::SectorNumber;
+  using primitives::address::Address;
+  using primitives::piece::PaddedPieceSize;
+  using primitives::piece::PieceInfo;
+  using primitives::piece::UnpaddedPieceSize;
+  using proofs::SealRandomness;
+  using sector_storage::InteractiveRandomness;
+  using sector_storage::Manager;
+  using sector_storage::PreCommit1Output;
+
+  struct SectorInfo {
```

Comment:
Is it the same is core/primitives/sector/sector.hpp:151?

Reply:
No, this is context object between FSM states